### ISSUE TITLE
Dangling migration in API

### ISSUE THREAD

**Issue by sarayourfriend** [WordPress/openverse#3310]
## Description

`just api/dj makemigrations` on `main` results in `api/api/migrations/0055_alter_matureimage_media_obj.py`.

## Additional context

Why didn't the `check_migrations` check catch this? The check is required :thinking: cc @dhruvkb in case you have any CI/CD related ideas. Theoretically it should be failing on all PRs that make changes to the API...

**Comment by dhruvkb** [comment]
Added a to-do for myself to investigate why the `checkmigrations` job did not catch this.

**Comment by sarayourfriend** [comment]
@dhruvkb Would you prefer to open a separate issue or to leave this one open after #3311 is merged?

**Comment by dhruvkb** [comment]
We can keep this open, I can't think the new one will have anything more to add that what's in the description of this issue.

**Comment by AetherUnbound** [comment]
Bumping this down to "high" priority since the original issue has been addressed.
